content/swiftui/concepts/views/terms/rectangle/rectangle.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Rectangle'
+Description: 'A rectangular shape aligned inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`rectangle`** A rectangular shape aligned inside the frame of the view containing it.
+
+## Syntax
+
+```pseudo
+var body: some View {
+  Rectangle()
+}
+```
+
+SwiftUI has several built-in shapes, such as `Rectangle`, circles, and capsules, and you can create, color, and place each shape as needed.
+
+## Example
+
+You can change it to the desired shape by attaching a modifier to the Rectangle.
```

Comment:
- In Docs, we refrain from using I, we, you, etc.
- Let's wrap rectangle in backticks -> `Rectangle`

Reply:
I changed it !